docs/introduction/vr-headsets-and-webxr-browsers.md

```diff
@@ -105,7 +107,7 @@ A-Frame supports include:
 
 Some other platforms that have been shown to work with A-Frame include:
 
-- Augmented reality (AR) on AR headsets (e.g., Magic Leap, HoloLens)
+- Augmented reality (AR) on AR headsets (e.g., Magic Leap, HoloLens, Apple Vision Pro)
```

Comment:
AVP does not belong on this list. Although definitions vary, I assume that "AR headsets" originally meant optical AR. Either way, when using WebXR it's not even possible to make use passthrough AR on the AVP (hopefully one day).